builtin/game/auth.lua

```diff
@@ -180,6 +187,20 @@ core.set_player_privs    = auth_pass("set_privileges")
 core.remove_player_auth  = auth_pass("delete_auth")
 core.auth_reload         = auth_pass("reload")
 
+function core.change_player_privs(name, changes)
+	local privs = core.get_player_privs(name)
+	for priv, change in pairs(changes) do
+		if change == true then
+			privs[priv] = true
+		elseif change == false then
+			privs[priv] = nil
+		else
+			core.log('warning', "non-bool value in `changes` table")
+		end
+	end
+	core.set_player_privs(name, privs)
+end
```

Comment:
honestly this is so trivial I'm wondering if we should even have it

Reply:
It's trivial and a big quality of life improvement -> definitely worth having